bundle/internal/schema/main.go

```diff
@@ -21,6 +21,19 @@ func addInterpolationPatterns(typ reflect.Type, s jsonschema.Schema) jsonschema.
 		return s
 	}
 
+	// The variables block in a target override allows for directly specifying
+	// the value if it is a primitive type.
+	if typ == reflect.TypeOf(variable.TargetVariable{}) {
+		return jsonschema.Schema{
+			AnyOf: []jsonschema.Schema{s,
+				{Type: jsonschema.StringType},
```

Comment:
I missed the `s` on line 28, please move it to its own line.

Reply:
done